modules/cats/src/main/scala/pureconfig/module/cats/package.scala

```diff
@@ -31,4 +32,10 @@ package object cats {
 
   implicit def nonEmptyVectorWriter[T](implicit vectorWriter: ConfigWriter[Vector[T]]): ConfigWriter[NonEmptyVector[T]] =
     ConfigWriter.fromFunction(nonEmptyVector => vectorWriter.to(nonEmptyVector.toVector))
+
+  implicit def nonEmptySetReader[T](implicit listReader: ConfigReader[SortedSet[T]]): ConfigReader[NonEmptySet[T]] =
```

Comment:
Nitpick: `listReader` is slightly misnamed. It's probably better called a `setReader`. Ditto on 39.